src/com/aca/homework/week2/projectuptime/ProjectTest.java

```diff
@@ -0,0 +1,27 @@
+package com.aca.homework.week2.projectuptime;
+
+import java.util.Scanner;
+
+public class ProjectTest {
+    public static void main(String[] args) {
+        long start = System.currentTimeMillis();
+        Project[] projects = new Project[3];
+        for (int i = 0; i < projects.length; i++) {
+            projects[i] = new Project();
```

Comment:
I think this line is unnecessary. you create a new Project then ignore it in next line.